src/main/java/dev/wakandaacademy/produdoro/tarefa/domain/Tarefa.java

```diff
@@ -55,4 +55,14 @@ public void pertenceAoUsuario(Usuario usuarioPorEmail) {
 			throw APIException.build(HttpStatus.UNAUTHORIZED, "Usuário não é dono da Tarefa solicitada!");
 		}
 	}
+
+	public void defineStatusAtivacao() {
+		this.statusAtivacao = StatusAtivacaoTarefa.ATIVA;
+	}
+
+	public void validacaoDeUsuario(UUID idUsuario) {
```

Comment:
muda esse metodo para private e use ele dentro desse metodo de ativa o satus ex: 

public void defineStatusAtivacao(UUID idUsuario) {
                validacaoDeUsuario(idUsuario)
		this.statusAtivacao = StatusAtivacaoTarefa.ATIVA;
	}